### PR TITLE
cannot submit a spike without a description and a title

### PR DESCRIPTION
**Pull request by kerrd89**
the else clause at this point is only console logging that your submission didn't work, we will need to build a ui component which lets the user know that their spike was not submitted and why

### CODE REVIEW

**Comment by kerrd89** [comment]
closes #40

File: lib/components/App.js [hunk 1]
```diff
@@ -59,18 +59,22 @@ export default class App extends Component {
...
+        appr: false,
+        count: 0,
+        createdAt: Date.now(),
+        createdBy: user.email,
+        attendees: []
+      };
+      reference.push(spike);
+      document.getElementById('ProposalForm').reset();
+    } else {
+      console.log('description and title required');
```
Comment (Jeff-Duke) [inline lib/components/App.js]: should we change this to an alert so the user is notified?

Comment (kerrd89) [inline lib/components/App.js]: updated!

**Comment by Jeff-Duke** [comment]
Looks good now.  Squashing and merging

**Pull request merged**